components/script_traits/lib.rs

```diff
@@ -596,6 +597,10 @@ pub struct InitialScriptState {
     pub webrender_document: DocumentId,
     /// FIXME(victor): The Webrender API sender in this constellation's pipeline
     pub webrender_api_sender: RenderApiSender,
+    // TODO pylbrecht
+    // write meaningful docstring
+    ///
+    pub layout_thread_is_busy: Arc<AtomicBool>,
```

Comment:
This probably isn't necessary given the earlier changes.

Reply:
I feel like storing the busy flag in `InitialScriptState` is the only way of passing it to argument to `ScriptThreadFactory::create()`. Or am I missing something here?